Templates/CSharp/Desktop/MSTestDesktopCSharp.csproj

```diff
@@ -29,6 +30,7 @@
     <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
     <IsVsixProj>true</IsVsixProj>
```

Comment:
In Arcade, the property appears to be `IsVsixProject`

https://github.com/dotnet/arcade/blob/9104c21492e1c6ed98105dd87388bca8f1f1abaa/src/Microsoft.DotNet.Arcade.Sdk/tools/VisualStudio.targets#L7

Reply:
As per this, I am guessing we don't even need that https://github.com/dotnet/arcade/blob/9104c21492e1c6ed98105dd87388bca8f1f1abaa/src/Microsoft.DotNet.Arcade.Sdk/tools/VisualStudio.props#L14